lib/naughty/core.lua

```diff
@@ -437,6 +438,11 @@ function naughty.notify(args)
     local text = args.text or preset.text
     local title = args.title or preset.title
     local s = get_screen(args.screen or preset.screen or screen.focused())
+    if not s then
+        require("gears.debug").print_warning(
+            "naughty.notify: there is no screen available to display the notification.")
```

Comment:
Yeah, makes sense after all!

(I've only thought about the `awesome.startup_errors` use case, where it gets printed already.)